src/main/kotlin/com/cout970/magneticraft/systems/tilerenderers/Utilities.kt

```diff
@@ -85,7 +85,9 @@ object Utilities {
                     }
 
                     scale(2.0, 2.0, 2.0)
-                    renderItemWithTransparency(stack, ItemCameraTransforms.TransformType.GROUND, 0.5f)
+                    if (Minecraft.getMinecraft().world.isAirBlock(ctx.center.plus(blockPos))) enableDepth()
+                    renderItemWithTransparency(stack, ItemCameraTransforms.TransformType.GROUND, 0.6f)
+                    disableDepth()
```

Comment:
PRO tip: you can use '+' instead of .plus()
In this case it would look like:
`if (Minecraft.getMinecraft().world.isAirBlock(ctx.center + blockPos)) enableDepth()`